src/develop/ui/patterns/mobile/columns.md

```diff
@@ -42,22 +47,24 @@ For an uneven number of columns, the following splits apply:
 
 ![](images/Column_break_first.png)
 
-## Input Parameters
+## Properties
 
-**Input Name** |  **Description** |  **Default Value**  
+**Property** |  **Description** |  **Default Value**  
 ---|---|---  
 ![](images/input.png) UseGutter  |  Creates a space between columns.  | True  
-![](images/input.png) PhonePortraitBreak  |  Behavior of the columns in a Phone with Portrait orientation.  |  BreakNone  
-![](images/input.png) PhoneLandscapeBreak  |  Behavior of the columns in a Phone with Landscape orientation.  |  BreakNone  
-![](images/input.png) TabletPortraitBreak  |  Behavior of the columns in a Tablet with Portrait orientation.  |  BreakNone  
-![](images/input.png) TabletLandscapeBreak  |  Behavior of the columns in a Tablet with Landscape orientation.  |  BreakNone  
+![](images/input.png) PhonePortraitBreak  |  Behavior of the columns on a phone with portrait orientation.  |  BreakNone  
```

Comment:
@annettewilliamsoutsystems we should outline the different values accepted, and what they mean. If you'd want to refer to the section where we describe the different breaks, and explain the different breaks there, that's also fine.